src/helpers/figures/charts/pie_chart.ts

```diff
@@ -225,6 +243,24 @@ function getPieColors(colors: ChartColors, dataSetsValues: DatasetValues[]): Col
   return pieColors;
 }
 
+function calculatePercentageValues(dataSetsValues) {
```

Comment:
Please type the argument and return values (even is using ChartJs types in annoying). Everything is typed as any, so you have some wrong types and nobody noticed. (data.map returning a single string? that's not how `map` works...)

And As stated in another comment, we can simplify _a lot_ the function to take a single dataset as argument, and return the percentage for a single value